utils/snapshot_extractor.mjs

```diff
@@ -30,7 +30,8 @@ export default async function snapshotExtractor(extractor, { inputs }) {
 
   let writeResult = "";
   const write = (string) => {
-    writeResult += string + "\n";
+    if (extractor.write) writeResult = extractor.write(writeResult, string);
```

Comment:
@TimDaub As you can see from the extractor code, I have used several workers to fetch all tokenIDs quickly. But the time each worker ends and write result is different, so I need part to get all results and put together. Of course, I will move parsing part on extractor, but I think this kind of part is needed.

Reply:
@Kunkka0822 I recommend you broadly and in much detail explain what you're trying to achieve in this pull request as I'm having trouble understanding what is actually going on here. E.g. why do you need to have the tokenID requests returned in order?